tests/RA_RichPresence_Tests.cpp

```diff
@@ -252,106 +257,109 @@ TEST_CLASS(RA_RichPresence_Tests)
     TEST_METHOD(TestConditionalDisplayNoDefault)
     {
         // A non-conditional string must be present
-        unsigned char memory[] = { 0x00, 0x12, 0x34, 0xAB, 0x56 };
-        InitializeMemory(memory, 5);
+        std::array<unsigned char, 5> memory{0x00, 0x12, 0x34, 0xAB, 0x56};
+        InitializeMemory(memory);
 
         RichPresenceInterpreterHarness rp;
         rp.LoadTest("Display:\n?0xH0000=0?Zero");
 
         Assert::AreEqual(false, rp.Enabled());
     }
-        
+
     TEST_METHOD(TestConditionalDisplaySharedLookup)
     {
-        unsigned char memory[] = { 0x00, 0x12, 0x34, 0xAB, 0x56 };
-        InitializeMemory(memory, 5);
+        std::array<unsigned char, 5> memory{0x00, 0x12, 0x34, 0xAB, 0x56};
+        InitializeMemory(memory);
 
         RichPresenceInterpreterHarness rp;
-        rp.LoadTest("Lookup:Location\n0x00=Zero\n0x01=One\n\nDisplay:\n?0xH0001=18?At @Location(0xH0000)\nNear @Location(0xH0000)");
+        rp.LoadTest(
+            "Lookup:Location\n0x00=Zero\n0x01=One\n\nDisplay:\n?0xH0001=18?At @Location(0xH0000)\nNear "
+            "@Location(0xH0000)");
 
         Assert::AreEqual("At Zero", rp.GetRichPresenceString().c_str());
 
-        memory[0] = 1;
+        memory.front() = 1;
         Assert::AreEqual("At One", rp.GetRichPresenceString().c_str());
 
-        memory[1] = 17;
+        memory.at(1) = 17;
         Assert::AreEqual("Near One", rp.GetRichPresenceString().c_str());
 
-        memory[0] = 0;
+        memory.front() = 0;
```

Comment:
prefer replacing all of these `memory.front()` calls with `memory.at(0)` for consistency